packages/api/endpoints/granules.js

```diff
@@ -124,15 +126,16 @@ async function put(req, res) {
       logger.info(`targetExecution has been specified for granule (${granuleId}) reingest: ${targetExecution}`);
     }
 
+    // FUTURE - this should not be part of the dynamo granule model
```

Comment:
These strike me as something we can handle in phase 3. We'll need to figure out what the role of the old models should be. Do they become "API" models? Are they eliminated altogether? Let me know if you disagree with leaving this for now.

Reply:
Agreed - Phase 2 scope is to make the postgres database the 'source of truth', not restructure/refactor this.     That said, there's no reason we shouldn't pull out logic from the models/refactor to an api `lib` (we've done this elsewhere) if it's ancillary to the data record storage/retrieval/transformation/etc, as *that* is consistent with the db model approach we're taking in potgres. 

In either case we don't need to address it in this PR.